functional-tests/functional_cases.h

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 IOTA Stiftung
```

Comment:
In `functional_cases.h` some sections of the code like structs, enums, function definition don't have a comments.